contracts/HubPool.sol

```diff
@@ -238,16 +243,19 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
      * @notice Whitelist an origin token <-> destination token route.
      */
     function whitelistRoute(
+        uint256 originChainId,
```

Comment:
Do you think adding `originChainId` to the function interface is the best way to get the L2 to know about the L2Token => destinationChainId route?